features/whitehall.feature

```diff
@@ -132,5 +132,4 @@ Feature: Whitehall
     And I force a varnish cache miss
     And I am benchmarking
     When I visit "/banknote"
-    # Redirects are transparently followed, so the end status code is 200
```

Comment:
I think I have to change it. For some reason, Poltergeist will successfully redirect pages, but won't update the status code from `301` to `200`. As pointed out [here|https://github.com/teamcapybara/capybara/issues/1149], integration tests like this should be using a website how a user would use a website - and that doesn't include looking at status codes. (Arguably better practice still than checking the path is checking for a known element on the page to ensure it's actually been rendered).

Why can't we change it?